client/src/components/Print.js

```diff
@@ -1,54 +1,58 @@
 import React, {useEffect, useState} from 'react'
+import { LazyLoadImage } from 'react-lazy-load-image-component'
+import 'react-lazy-load-image-component/src/effects/opacity.css';
+import ImageFallbacks from './layout/ImageFallbacks';
 
 const Print = (props) => {
     // Function Props
-    const { incrementLoaded, open } = props
+    const { open } = props
     // Variable Props
-    const { src, title, sku, price, sold, size, original, loaded, position } = props
+    const { src, title, sku, price, sold, size, original, position } = props
 
     const [isZero, setIsZero] = useState(false)
     const [inCart, setInCart] = useState(false)
-
-    var savedCart = JSON.parse(localStorage.getItem("cart"));
+    const [loaded, setLoaded] = useState(false)
+    const [brokenLink, setBrokenLink] = useState(false)
 
     useEffect(() => {
         sold && setIsZero(true)
     }, [sold])
 
+    var savedCart = JSON.parse(localStorage.getItem("cart"));
+
     useEffect(() => {
-        savedCart?.find(item => {
-            if(item.original && item._id === sku) {
-                setInCart(true)
-            }
-            return null
-        })
+        var foundItem = savedCart?.find(item => item.original && item._id === sku)
+        if (foundItem) setInCart(true)
```

Comment:
Searching by id here, don't need to check if item is original

Reply:
I do need to check as this is setting the inCart state depending on whether or not the image is one of a kind.